vars/runGradleCheck.groovy

```diff
@@ -71,7 +78,7 @@ void call(Map args = [:]) {
 
                 echo "Start gradlecheck"
                 GRADLE_CHECK_STATUS=0
-                ./gradlew clean && ./gradlew check -Dtests.coverage=true --no-daemon --no-scan || GRADLE_CHECK_STATUS=1
+                ./gradlew clean && ./gradlew check -Dtests.coverage=true ${bwc_checkout_align_param} --no-daemon --no-scan || GRADLE_CHECK_STATUS=1
```

Comment:
Does the below suggestion make it less confusing?

```suggestion
                ./gradlew clean && ./gradlew check -Dtests.coverage=true -Dbwc.checkout.align=${bwc_checkout_align} --no-daemon --no-scan || GRADLE_CHECK_STATUS=1
```

Reply:
I am setting the value of `bwc_checkout_align` to `'-Dbwc.checkout.align=true'`. See tests.

Reply:
Just in case the default value changes in OpenSearch repo, it will be easier to debug using the command in the gradle check run. Your call

Reply:
I feel like at the moment we can keep it removed if false, since it is the origin behavior unchanged.